app/src/main/java/org/systers/mentorship/utils/Constants.kt

```diff
@@ -7,4 +7,6 @@ object Constants {
     const val REQUEST_LIST = "request_list"
     const val REQUEST_EMPTY_LIST_TEXT = "request_empty_list_text"
     const val RELATIONSHIP_EXTRA = "relationship_extra"
+    const val DELETE_REQUEST_RESULT_ID = 1000
+    const val DELETE_REQUEST_RESULT_VALUE = "requestid"
```

Comment:
Good job @shiv07tiwari :+1: 
But one more change here, I think appropriate name for "requestid" would be **const val REQUEST_ID.**
As you can see the naming conventions for the other constants as well. What do you think @isabelcosta ?

Reply:
Yes that would be appropriate @mohak1283 and @shiv07tiwari